src/nagendra/JavaArray.java

```diff
@@ -0,0 +1,84 @@
+package nagendra;
+
+public class JavaArray {
+
+	int getMinimumNumber(int[] num) {
+
+		int minimum = num[0];
+
+		for (int index = 0; index < num.length; index++) {
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files